apps/ledger-live-mobile/src/context/LedgerStore.tsx

```diff
@@ -70,24 +70,29 @@ export default class LedgerStoreProvider extends Component<
     const possibleIntermediaries = [bitcoin, ethereum];
 
     const getsupportedCountervalues = async () => {
-      const supportedFiats = await listSupportedFiats();
-      const supportedCounterValues = [...supportedFiats, ...possibleIntermediaries]
+      const res = await listSupportedFiats();
```

Comment:
I think that `supportedFiats` was a better variable name than `res`, is there a reason for this change that I don't see?